src/main/java/net/rcarz/jiraclient/tempo/Team.java

```diff
@@ -0,0 +1,178 @@
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSummary() {
+        return summary;
+    }
+
+    public void setSummary(String summary) {
+        this.summary = summary;
+    }
+
+    public String getLead() {
+        return lead;
+    }
+
+    public void setLead(String lead) {
+        this.lead = lead;
+    }
+
+    public static List<Team> get(RestClient restclient) throws JiraException {
+        JSON response = null;
+
+        try {
+            response = restclient.get(getRestUri());
+        } catch (Exception ex) {
+            throw new JiraException("Failed to retrieve all teams", ex);
+        }
+
+        if (!(response instanceof JSONArray)) {
+            throw new JiraException("JSON payload is malformed");
+        }
+        List<Team> result = new ArrayList<Team>();
+        Iterator it = ((JSONArray) response).iterator();
+        while (it != null && it.hasNext()) {
+            result.add(new Team((JSONObject) it.next()));
+        }
+
+        return result;
+    }
+
+    public static Team get(RestClient restClient, Integer id) throws JiraException{
+        JSON response = null;
+
+        try {
+            response = restClient.get(getRestUri() + id.toString());
+        } catch (RestException rx) {
+            if (rx.getHttpStatusCode() == HttpStatus.SC_NOT_FOUND) {
+                return null;
+            }
+        } catch (Exception ex) {
+            throw new JiraException("Failed to retrieve team by id: " + id, ex);
+        }
+
+        if (!(response instanceof JSONObject)) {
+            throw new JiraException("JSON payload is malformed");
+        }
+        JSONObject result = (JSONObject) response;
+        if (!result.containsKey(ID) || !(result.get(ID) instanceof Number)) {
+            return null;
+        }
+
+        return new Team(result);
+    }
+
+    public static Team create(RestClient restClient, String name, String summary, String lead) throws JiraException {
+        JSON result = null;
+
+        try {
+            result = restClient.post(getRestUri(), new Team(null, name, summary, lead).asJsonObject());
+        } catch (Exception ex) {
+            throw new JiraException("Failed to create team", ex);
+        }
+
+        if (!(result instanceof JSONObject) || !((JSONObject) result).containsKey(ID)
+                || !(((JSONObject) result).get(ID) instanceof Number)) {
+            throw new JiraException("Unexpected result on team creation: " + result.toString());
+        }
+
+        return new Team((JSONObject) result);
+    }
+
+    public static boolean delete(RestClient restClient, Integer id) throws JiraException {
+        JSON result = new JSONObject();
+
+        if (get(restClient, id) != null) {
```

Comment:
Value of performance is higher than exact boolean result. So use just delete method. My note was just about code:
delete(rest, 3);
delete(rest, 3);
should not give exception that "3 doesn't exist" at second call.